### PR TITLE
Updated readme, pointing to website

### PR DESCRIPTION
**Pull request by rustyrazorblade**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,14 @@ Reaper comes with a GUI, which if you're running in local mode can be at http://
 Please see the [Issues](https://github.com/thelastpickle/cassandra-reaper/issues) section for more
 information on planned development, and known issues.
 
+Documentation and Help
+------------------------
+
+The full documentation is available at the [Reaper website](http://cassandra-reaper.io/).  The [source for the website](https://github.com/thelastpickle/reaper-site) is available as well, and pull requests are encouraged!
+
+
+Have a question?  Please ask on the [reaper mailing list](https://groups.google.com/forum/#!forum/tlp-apache-cassandra-reaper-users)!  We also have a chat
```
Comment (michaelsembwever) [inline README.md:28]: are you missing the link and a full-stop at the end of this line?

**Comment by michaelsembwever** [comment]
LGTM. Can you squash those 3 commits and write a "proper" commit message please.

File: README.md [hunk 2]
```diff
@@ -19,6 +19,14 @@ Reaper comes with a GUI, which if you're running in local mode can be at http://
 Please see the [Issues](https://github.com/thelastpickle/cassandra-reaper/issues) section for more
 information on planned development, and known issues.
 
+Documentation and Help
+------------------------
+
+The full documentation is available at the [Reaper website](http://cassandra-reaper.io/).  The [source for the website](https://github.com/thelastpickle/reaper-site) is available as well, and pull requests are encouraged!
```
Comment (ossarga) [inline README.md:25]: Minor
Does the sentence sound ok if we add *there* into it?

> ... is available there as well, and ....

**Comment by ossarga** [comment]
LGTM. Same as what @michaelsembwever said about squashing and commit message.

**Pull request closed without merging**